lib/read/test/Test/Unit/Cardano/Read/Ledger/Tx.hs

```diff
@@ -142,6 +143,34 @@ maryTx = unsafeParseEraTxFromHex
     \f79d4c93f29871fc51b78aeba80e58200000000000000000000000\
     \00000000000000000000000000000000000000000044a1024100f6"
 
+-- Example of a transaction for Mary that involves an issue
+-- with the ledger where the end of overlong addresses
+-- is discarded.
```

Comment:
What kind of overlong address is this?

Reply:
I was under the impression that it's a base address whose binary serialization does not match the original bytes in this transaction — the bytes are truncated when parsing.

Reply:
👌 this seems to be the address with the highlighted part being the stake credential
<img width="745" alt="Skärmavbild 2024-08-23 kl  16 55 29" src="https://github.com/user-attachments/assets/75352694-8ceb-43c6-b094-98c7108f7e19"> 😮

 Is this on mainnet? Where did you find this?

Reply:
Yeah, this is on mainnet:

https://cardanoscan.io/transaction/369826b2f60b443d27966d222fa5bedc6b900326b77a09f15473d77f5ef2871c?tab=utxo

It was brought up today in the ledger channel, and I figured that I should record the finding here in order to better preserve it.

Reply:
(I have amended the comment in the source code to better explain this.)